### ISSUE TITLE
Can't build Docker image based on Ubuntu 17.10

### ISSUE THREAD

**Issue by rod-glover** [pacificclimate/pdp#122]
Current Dockerfile specifies `FROM ubuntu:17.10`. 

Image builds based on this fail; investigation reveals that [Ubuntu 17.10 apparently doesn't exist or at least isn't supported](https://wiki.ubuntu.com/Releases).

Updating this to  `FROM ubuntu:18.04` enables images to be built.

**Comment by jameshiebert** [comment]
Yeah, it was EOL last summer. Guess we never noticed, b/c it's cached on most of our machines:

```
hiebert@docker-dev01:~$ docker images | grep ubuntu
ubuntu                                                                                17.10               275d79972a86        18 months ago       94.6MB
...
```

**Comment by rod-glover** [comment]
Shall we upgrade? Is there a risk there?

**Comment by jameshiebert** [comment]
The only risk I'd see is the GDAL minor version changing. If it's static, go for it. If it's not, probably want to run the full, hardcore test suite against all the data.

**Comment by jameshiebert** [comment]
This was fixed in 31432aa5
